### PR TITLE
chore: add category descriptions

### PR DESCRIPTION
**Pull request by Tlacenka**
In order to showcase the category descriptions on portal, I added examples in our CLI configuration.

### CODE REVIEW

File: code-pushup.config.ts [hunk 1]
```diff
@@ -87,18 +87,24 @@ const config: CoreConfig = {
     {
       slug: 'bug-prevention',
       title: 'Bug prevention',
+      description:
+        'Bug prevention category consists of rules and checks that are essential for stability and functionality of your codebase. It currently contains [ESLint](https://eslint.org/) rules that indicate potential bugs in your codebase.',
```
Comment (matejchalk) [inline code-pushup.config.ts:91]:
```suggestion
      description: 'Lint rules that find potential bugs in your code.',
```

File: code-pushup.config.ts [hunk 2]
```diff
@@ -87,18 +87,24 @@ const config: CoreConfig = {
...
       refs: [{ type: 'group', plugin: 'eslint', slug: 'problems', weight: 1 }],
     },
     {
       slug: 'code-style',
       title: 'Code style',
+      description:
+        'Code style category contains rules and guidelines that promote good practices and consistency in your code. ',
```
Comment (matejchalk) [inline code-pushup.config.ts:98]:
```suggestion
      description: 'Lint rules that promote good practices and consistency in your code.',
```

File: code-pushup.config.ts [hunk 3]
```diff
@@ -87,18 +87,24 @@ const config: CoreConfig = {
...
       refs: [
         { type: 'group', plugin: 'eslint', slug: 'suggestions', weight: 1 },
       ],
     },
     {
       slug: 'code-coverage',
       title: 'Code coverage',
+      description:
+        'Code coverage measures how much your application code is covered by tests.',
```
Comment (matejchalk) [inline code-pushup.config.ts:107]:
```suggestion
      description: 'Measures how much of your code is covered by tests.',
```

**Review by matejchalk — COMMENTED** [review]
Added suggestions on how to make the descriptions more concise and to the point.

File: code-pushup.config.ts (hunk 1 above)
Comment (Tlacenka) [reply code-pushup.config.ts:91]: I wondered about this. Shouldn't we include the general idea of a category to showcase as well? Although on this project it doesn't matter as much, I suppose.

File: code-pushup.config.ts [hunk 4]
```diff
@@ -111,6 +115,7 @@ const config: CoreConfig = {
     {
       slug: 'security',
       title: 'Security',
+      description: 'Finds known vulnerabilities in 3rd-party packages.',
```
Comment (matejchalk) [inline code-pushup.config.ts:118]:
```suggestion
      description: 'Finds known **vulnerabilities** in 3rd-party packages.',
```

**Review by matejchalk — DISMISSED** [review]
Maybe highlighting keywords would help non-techies to understand? Added some suggestions, but I leave it up to you.

**Pull request merged**